framework/src/main/java/org/eu/droid_ng/wellbeing/framework/WellbeingFrameworkServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package org.eu.droid_ng.wellbeing.framework;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.RemoteException;
+import android.os.UserHandle;
+import android.provider.Settings;
+
+import org.eu.droid_ng.wellbeing.lib.IWellbeingFrameworkService;
+
+public class WellbeingFrameworkServiceImpl extends IWellbeingFrameworkService.Stub {
+    private UserHandle UserHandle_ALL;
+    private final Context context;
+
+    @SuppressWarnings("JavaReflectionMemberAccess")
```

Comment:
Non reflection version for inside ROMs?

Reply:
So I do a shim/real system like for the main app?

Reply:
yes